docs/REST API/rest-api.md

```diff
@@ -8979,49 +8986,156 @@ Request (using `filters`):
 }
 ```
 
-### Get batch script summary
+### List batch scripts
 
-Get statuses and host counts for a batch-run script.
+Returns a list of batch script executions.
 
-`GET /api/v1/fleet/scripts/batch/summary/:batch_execution_id`
+`GET /api/v1/fleet/scripts/batch`
 
 #### Parameters
 
 | Name            | Type    | In   | Description                                                                                    |
 | ----            | ------- | ---- | --------------------------------------------                                                   |
-| batch_execution_id | string | path | **Required**. The ID returned from a batch script run. |
+| team_id         | integer | query | _Available in Fleet Premium_. Filters to batch script runs for the specified team. |
+| status          | string  | query | Filters to batch script runs with this status. Either `"started"`, `"scheduled"`, or "`completed`". |
+| page            | integer | query | Page number of the results to fetch. |
+| per_page        | integer | query | Results per page. |
 
 
 #### Example
 
-`GET /api/v1/fleet/scripts/batch/summary/abc-def`
+`GET /api/v1/fleet/scripts/batch`
 
 ##### Request body
 
 ```json
 {
-  "batch_execution_id": "e797d6c6-3aae-11ee-be56-0242ac120002"
+  "team_id": 123,
+  "status": "completed"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "batch_executions": [
```

Comment:
Realized we want `batch_execution_id` returned here too because we'll need it when clicking on an item in the batch scripts list, to transition to the batch script details.

Reply:
Will it always be non-null?